graphpype/tests/test_utils_plot.py

```diff
@@ -0,0 +1,113 @@
+
+"""test over plots (with matplotlib)"""
+
+import os
+import shutil
+import numpy as np
+
+from graphpype.utils_plot import (plot_cormat, plot_ranged_cormat,
+                                  plot_int_mat, plot_hist, plot_signals,
+                                  plot_sep_signals)
+
+tmp_dir = "/tmp/test_graphpype"
+nb_nodes = 10
+cor_mat = np.random.rand(nb_nodes, nb_nodes)
+int_mat = np.random.randint(low=-4, high=4, size=(nb_nodes, nb_nodes))
+nb_timepoints = 1000
+signals_matrix = np.random.rand(nb_nodes, nb_timepoints)
+
+try:
+    shutil.rmtree(tmp_dir)
+except OSError:
+    print("dir {} do not already exists".format(tmp_dir))
+
+try:
+    os.makedirs(tmp_dir)
+except OSError:
```

Comment:
you should avoid `try-except` when you can ... it hides bugs.

Here for instance, you should use `os.path.exists`.